src/detectors/BHCAL/BHCAL.cc

```diff
@@ -12,10 +12,66 @@
 #include "factories/calorimetry/CalorimeterClusterRecoCoG_factory.h"
 #include "factories/calorimetry/CalorimeterHitDigi_factory.h"
 #include "factories/calorimetry/CalorimeterHitReco_factory.h"
+#include "factories/calorimetry/CalorimeterHitsMerger_factory.h"
 #include "factories/calorimetry/CalorimeterIslandCluster_factory.h"
 #include "factories/calorimetry/CalorimeterTruthClustering_factory.h"
 #include "factories/calorimetry/TrackClusterMergeSplitter_factory.h"
 
+namespace eicrecon {
+
+  // --------------------------------------------------------------------------
+  // Helper method to generate appropriate phi map
+  // --------------------------------------------------------------------------
+  std::string MakePhiMap(const std::size_t nMerge = 1) {
```

Comment:
Fair! Part of adding this was just to provide an example of how to wire this into a complete workflow, but these can be easily moved downstream into some analysis scripts for the BHCal DSC.

(Though it would be pretty cool to be able to provide std::function's as parameters 👀)